pubspec.yaml

```diff
@@ -8,10 +8,10 @@ author: Devon Carew <devoncarew@google.com>
 environment:
   sdk: '>=1.10.0 <2.0.0'
 dependencies:
-  args: '^0.13.1'
-  logging: '^0.11.0'
+  args: '^0.13.2'
+  logging: '^0.11.1'
```

Comment:
Makes sense. Many of the problems I've run into w/ pub have involved incompatible version constraint windows. It's pretty frustrating when you have two packages you depend on that specify separate, incompatible version subranges...